Source/CesiumRuntime/Private/CesiumSunSky.cpp

```diff
@@ -254,6 +260,39 @@ void ACesiumSunSky::Tick(float DeltaSeconds) {
   if (this->UpdateAtmosphereAtRuntime) {
     this->UpdateAtmosphereRadius();
   }
+
+  if (IsValid(this->SkyAtmosphere)) {
+    double scale = this->_computeScale();
```

Comment:
Should we clamp to a minimum sensible value here? 
(Ex. Actor scale gets set to 0, we eventually divide by that in this function)

Reply:
Hmm not sure where that came from, but it shouldn't be needed. There's a similar check in `Tick`, so no need to do anything in particular in `BeginPlay`. Removed.

Reply:
Did you make a change to this logic? I saw you removed the blank if statement, but this double on line 265 is still around, and can still be bogus if the actor scale is bogus.

Reply:
Ah yeah thanks for keeping on me. My comment above was meant to go on the "blank if statement". I somehow skipped over this one.

I think this is ok, though. If we set the scale to zero, the atmosphere height and other things will become zero, but that's arguably correct and it doesn't seem to do any harm. The alternative would be to impose an arbitrary limit on the scale. We've had to do that elsewhere, but it's better not to do that when we can avoid it (as we can here).

Reply:
Fair enough, thanks for the explanation.